src/ssort/_main.py

```diff
@@ -39,7 +39,12 @@ def main():
     for path in find_python_files(args.files):
         errors = False
 
-        original_bytes = path.read_bytes()
+        try:
+            original_bytes = path.read_bytes()
+        except FileNotFoundError:
+            sys.stderr.write(f"ERROR: {str(path)!r} does not exist\n")
+            unsortable += 1
+            continue
```

Comment:
Probably also doing `IsADirectoryError` and `PermissionError` in the same batch.  I don't think there are any other errors which are expected.  `IsADirectoryError` will be a pain to test as I think it can only result from a race condition.

Reply:
Added. I skipped adding tests for IsADirectoryError, however. I image you could use file locking to consistently test it but the complexity of such a test doesn't seem worth it for such a rare edge case.